.travis.yml

```diff
@@ -51,7 +51,8 @@ jobs:
         # Your test script goes here
         - echo ">>> Run tests"
         # use XVFB to have headless display port, and still run the Matplotlib tests.
-        - xvfb-run -a pytest --cov=./ --durations=10
+        - xvfb-run -a pytest -m "fast" --cov=./ --durations=10
```

Comment:
@minouHub I think that by using this you might have replaced the default configuration which is defined in [setup.cfg](https://github.com/radis/radis/blob/develop/setup.cfg#L13) , and which has this value : 

```
-m "not needs_cuda and not download_large_databases and not needs_db_CDSD_HITEMP and not needs_db_CDSD_HITEMP_PCN and not needs_db_CDSD_HITEMP_PC and not needs_db_HITEMP_CO2_DUNHAM and not needs_db_HITEMP_CO_DUNHAM"
```
The value is quite long, but it explicitely discards some tests, including the `not needs_db_CDSD_HITEMP_PC`  that require databases that are not in the test files. These tests are responsible for the current fails in https://github.com/radis/radis/pull/647

I'd suggest to keep : 

```
        - xvfb-run -a pytest -m "fast" --durations=10  # no "fast" tests use the local databases 
        - xvfb-run -a pytest --cov=./ --durations=10  # uses default parameters in setup.cfg
 ```

Reply:
oh, I see! By default, all the tests you quote are skipped, but when I mention `-m "fast"` it overrides the default... Thanks for looking at it